src/curves.py

```diff
@@ -145,6 +145,49 @@ def _post_process(curve_string: str, replacements: dict) -> str:
   return pattern.sub(lambda x: replacements[x.group(0)], curve_string)
 
 
+class Hendragon2(Curve):
+
+  def __init__(self, size=1000, start = '', width=3):
+    rules = \
+      "M -> MRrLllr;" + \
+      "r -> MrRMLlr;" + \
+      "R -> MrrRlLr;" + \
+      "l -> rRLlllr;" + \
+      "Ll-> rRLlllMlRrLllr;"+ \
+      "Lr-> rRLlllMrLlRrrl;"+ \
+      "LR-> rRLlllMMLRrrrl;"
+
+    self.lsys = ls.LSystem(rules, start_symbol='rrrrRLR')
```

Comment:
start_symbol should maybe be a single Variable

Reply:
One can also use `M` as a start symbol. However, I would rather use the current start_symbol, because it produces a closed curve, which cannot be done with one symbol.